fineract-loan/src/main/java/org/apache/fineract/portfolio/loanaccount/data/LoanSummaryData.java

```diff
@@ -91,9 +91,14 @@ public class LoanSummaryData {
     private BigDecimal totalCreditBalanceRefundReversed;
     private BigDecimal totalRepaymentTransaction;
     private BigDecimal totalRepaymentTransactionReversed;
+    private BigDecimal totalInterestRefund;
+    private BigDecimal totalInterestPaymentWaiver;
     private final Long chargeOffReasonId;
     private final String chargeOffReason;
 
+    private BigDecimal totalUnpaidAccruedDueInterest;
```

Comment:
Fields were created, but i dont see where these are calculated and sent back in the response!